bgpd/bgp_attr.c

```diff
@@ -1603,11 +1603,12 @@ enum bgp_attr_parse_ret bgp_attr_nexthop_valid(struct peer *peer,
 					       struct attr *attr)
 {
 	in_addr_t nexthop_h;
+	struct bgp *bgp = peer->bgp;
 
 	nexthop_h = ntohl(attr->nexthop.s_addr);
-	if ((IPV4_NET0(nexthop_h) || IPV4_NET127(nexthop_h)
-	     || IPV4_CLASS_DE(nexthop_h))
-	    && !BGP_DEBUG(allow_martians, ALLOW_MARTIANS)) {
+	if ((IPV4_NET0(nexthop_h) || IPV4_NET127(nexthop_h) ||
+	     IPV4_CLASS_DE(nexthop_h)) &&
+	    !bgp->allow_martian) {
```

Comment:
We discussed this in the tech meeting and the decision was made to remove this immediately since there was no way anyone was reasonably using this due to the way it was coded.

Reply:
and finally I'm dubious about your claim about fixing the labN ci failure by putting this back.

Reply:
I think igor is right...

Reply:
... but will change labn ci to use both old and new config line

Reply:
@donaldsharp what are you dubious about? The CI was using `debug bgp allow-martians` and now it doesn't work. If you make it work again, it should fix the CI.

@louberger the new command is not global, you have to put it inside every `router bgp` that you configure in your tests.